docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-subcases-con.adoc

```diff
@@ -0,0 +1,15 @@
+[id='case-management-subcases-proc_{context}']
+= Creating sub cases in case management projects
+
+Subcases give you the flexibility to compose advanced cases that consists of other cases. This means that you can split large and complex cases into multiple layers of abstraction and even multiple case projects. This is similar to splitting a process into multiple subprocesses.
+
+Case definition is set of activities that might happen while the case instance is active. It is completely dynamic and therefore, what is in the definition are not always the only activities that can be invoked in the case instance. Dynamic activities can also be included during run time on an already active case instance.
+
+A case instance is a single instance of a case definition and should encapsulate the given business context. All case instance data is stored in the case file, which is then accessible to all process instances that might participate in the given case instance. Each case instance is completely isolated from the other and the same applies for the case file. Only owners of the case instance can access the case file.
```

Comment:
>  Only owners of the case instance can access the case file.

I'd change this to " Only participants of the case instance can access the case file."

Reply:
Done. Changed to "Only case instance participants can access the case file."